docs/volume-management.md

```diff
@@ -0,0 +1,150 @@
+  finalizers:
+  - kubernetes.io/pvc-protection
+  labels:
+    app: minio
+  name: minio-data-1-minio-0
+  namespace: default
+  resourceVersion: "76360"
+  uid: d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 16Mi
+  storageClassName: directpv-min-io
+  volumeMode: Filesystem
+  volumeName: pvc-d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+status:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 16Mi
+  phase: Bound
+
+# Edit 'minio-data-1-minio-0' PVC to increase the size from 16MiB to 64MiB.
+$ cat minio-data-1-minio-0.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    pv.kubernetes.io/bind-completed: "yes"
+    pv.kubernetes.io/bound-by-controller: "yes"
+    volume.beta.kubernetes.io/storage-provisioner: directpv-min-io
+    volume.kubernetes.io/selected-node: master
+    volume.kubernetes.io/storage-provisioner: directpv-min-io
+  creationTimestamp: "2023-06-08T04:46:02Z"
+  finalizers:
+  - kubernetes.io/pvc-protection
+  labels:
+    app: minio
+  name: minio-data-1-minio-0
+  namespace: default
+  resourceVersion: "76360"
+  uid: d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 64Mi   # <--- increase size here
+  storageClassName: directpv-min-io
+  volumeMode: Filesystem
+  volumeName: pvc-d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+status:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 16Mi
+  phase: Bound
+
+# Apply changes
+$ kubectl apply -f minio-data-1-minio-0.yaml
+
+# After successful expansion, you will see updated YAML
+$ kubectl get pvc minio-data-1-minio-0 -o yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    pv.kubernetes.io/bind-completed: "yes"
+    pv.kubernetes.io/bound-by-controller: "yes"
+    volume.beta.kubernetes.io/storage-provisioner: directpv-min-io
+    volume.kubernetes.io/selected-node: master
+    volume.kubernetes.io/storage-provisioner: directpv-min-io
+  creationTimestamp: "2023-06-08T04:46:02Z"
+  finalizers:
+  - kubernetes.io/pvc-protection
+  labels:
+    app: minio
+  name: minio-data-1-minio-0
+  namespace: default
+  resourceVersion: "76651"
+  uid: d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 64Mi
+  storageClassName: directpv-min-io
+  volumeMode: Filesystem
+  volumeName: pvc-d7fad69a-d267-43c0-9baf-19fd5f65bdb5
+status:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 64Mi # <--- increased size here
+  phase: Bound
+```
+
+## Delete volume
+***CAUTION: THIS IS DANGEROUS OPERATION WHICH LEADS TO DATA LOSS***
+
+Volume can be deleted only if it is in `Ready` state (that is, no pod is using it). Run the `kubectl delete pvc` command which triggers DirectPV volume deletion. As removing a volume leads to data loss, double check what volume you are deleting. Below is an example:
+```sh
```

Comment:
can we also explain about `kubectl directpv clean` command which can be used to clean state volumes? or volumes of the pods which were force deleted.

Reply:
I will add another section `## Clean stale volumes`.